### PR TITLE
Horn

### PR DESCRIPTION
**Pull request by Achmiel4**
Component created for the horn oscillator.

### CODE REVIEW

**Comment by KyungSun4** [comment]
You should resolve the merge conflicts by merging master into your branch

**Comment by KyungSun4** [comment]
I am assuming the actual change in this which should be more clear after you merge master, is IRFR010.
I know little about symbol design however, if you compare this symbol to the other mosfets, I would sugest adding keywords N-Channel MOSFET, turning off show pin names and place pin names inside. and italicize the footprint and left align everything but the footprint. to make these changes go to the symbol editor and edit symbol properties.

File: schematic_kicad/Transistor_FET.dcm [hunk 1]
```diff
@@ -1404,6 +1422,18 @@ K Dual N-Channel P-Channel MOSFET
 F https://www.fairchildsemi.com/datasheets/FD/FDS9934C.pdf
 $ENDCMP
 #
+$CMP IPD50R380CE
```
Comment (KyungSun4) [inline schematic_kicad/Transistor_FET.dcm:1425]: I'm not sure what happend but i'm guessing these changes shouldn't be here

**Review by rally314 — CHANGES_REQUESTED** [review]
It looks like your schematic is referencing the rescue library of your symbol (at the bottom of the nickname section) instead of the library path of our submodule.

The rescue file is basically like a saved copy of the symbol if the symbol is deleted accidentally. For us, rescue files usually generate because the submodule might not up to date and that could create a temp file for your schematic which isn't able to be modified by other ppl because usually only you have it on your comp. I don't have the rescue on my comp but it looks like KiCad is trying to access it. 

Make sure to delete the rescue file and ensure the referenced symbol is in the libraries folder.

image

**Review by jtmullen — CHANGES_REQUESTED** [review]
There are a lot of weird things in here that I think are probably all unintentional. It is easy to accidentally copy things in the symbol editor and it will save them all. 

> I am assuming the actual change in this which should be more clear after you merge master, is IRFR010.

If that is the case, I don't think this is needed? `device:Q_NMOS_GDS` is the right symbol for it.

File: schematic_kicad/Transistor_FET.dcm (hunk 1 above)
Comment (Achmiel4) [reply schematic_kicad/Transistor_FET.dcm]: It seems like things got moved around and copied unintentionally, I manually removed the copies and moved some things it should be good now, the only thing that is relevant is IRFR010.

**Comment by Achmiel4** [comment]
@rally314 I removed the rescue library from the Project Specific Libraries as well as deleting the file from the HornOsc_v0.1 folder, but I think those changes aren't on this submodule.

**Comment by jtmullen** [comment]
Also @Achmiel4 you shouldn't be changing anything in the mirrored kicad libraries, only our ISC libraries. You can see this repos wiki page for more info

**Comment by Achmiel4** [comment]
@jtmullen would it be better to just scrap the new symbol and this pull request and just use the Q_NMOS_DGS symbol on the schematic?

**Comment by jtmullen** [comment]
> @jtmullen would it be better to just scrap the new symbol and this pull request and just use the Q_NMOS_GDS symbol on the schematic?

That would be my recommendation yes. I don't see any reason that part can't use a standard mosfet symbol

**Comment by rally314** [comment]
Oh huh. Yeah thats weird that it was created before. You can def use the q_nmos_gds symbol for that

**Pull request closed without merging**